uber/templates/jobs/staffers.html

```diff
@@ -70,7 +70,9 @@
 <div style="margin-left:10px">
     [D]: <i>Staffers marked with a "D" are department heads.</i> <br/>
     [M]: <i>Staffers marked with a "M" are assigned to multiple locations</i> <br/>
-    [T]: <i>Staffers marked with a "T" are trusted</i> <br/>
+    [T]: <i>Staffers marked with a "T" are trusted
+         {% if location %}in this department{% else %}in at least one department{% endif %}
+         </i> <br/>
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
```

Comment:
Yeah, this is fine. I think it's obvious from context that [T] would mean "trusted in some unnamed department(s)."